packages/components/src/components/table/style.css

```diff
@@ -25,6 +25,12 @@
 	div.pagination {
 		order: 2;
 	}
+    
+	div.pagination kol-pagination  {
```

Comment:
@sdvg Bitte Feedback zur Änderung am Basis-Style. Passt das?

Reply:
@deleonio @cyrinermd Ich habs mir jetzt mal lokal ausgecheckt und genauer angeschaut:

* Das Padding würde ich wieder raus nehmen, gehört m.M.n. nicht ins Basis Style sondern bei Bedarf in die Themes
* Können wir diesen "Breakpoint" noch  optimieren? Das ist bei 1024vw und ich fände es besser, wenn nach dem "Einträge 11 bis 20 von 100 angezeigt" schon umgebrochen würde.
<img width="762" alt="image" src="https://github.com/public-ui/kolibri/assets/971072/bec0c2ee-7fb6-4583-a7ee-8561a2bc5f10">